manifests/profile/dns/aws.pp

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::dns::standard
+#
+# Set up standard resolv_conf.
+#
+# @example
+#   include nebula::profile::dns::aws
+class nebula::profile::dns::aws {
+
+  $searchpaths = lookup('nebula::resolv_conf::searchpath')
+  $searchpath_str = join($searchpaths, ' ')
+
+  exec { 'restart_networking':
+    command     => '/bin/systemctl restart networking',
+    refreshonly => true,
+  }
+
+  file_line {
+    default:
+      path   => '/etc/dhcp/dhclient.conf',
+      notify => Exec['restart_networking'],
+    ;
```

Comment:
I don't recognize this syntax. What's going on with the `default:` and the dangling `;`?

Reply:
Ahh, thanks. "Full syntax" looks a little wacky to me. I can go either way, but here's an alternative for consideration (containing a more explicit resource title and using two resources since the default bits are simple and locality is very high):

```
file_line { 'DHCP local search domain':
  path   => '/etc/dhcp/dhclient.conf',
  notify => Exec['restart_networking'],
  line => "supersede domain-search \"${searchpath_str}\";",
}

file_line { 'DHCP local domain name':
  path   => '/etc/dhcp/dhclient.conf',
  notify => Exec['restart_networking'],
  line => 'supersede domain-name "";',
}
```

Reply:
I think it only looks wacky because we haven't used it much before. I think iit makes a lot of sense to use this syntax here.